registry.json

```diff
@@ -1040,6 +1040,21 @@
     "operatorName": "Oleh",
     "github": "olegandreyev"
   },
+  {
+    "ticker": "MADT",
+    "poolId": "d7876f8358f5e479ae2839de62d43353b07b2006a40fbf94e723c18c",
```

Comment:
Your pool id is : d7876f8358f5e479ae2839de62d43353b07b2006a40fbf94e723c18c

Reply:
yes it is, isn't it ?